src/app/static/src/app/store/downloadResults/downloadResults.ts

```diff
@@ -0,0 +1,43 @@
+import {RootState} from "../../root";
+import {DownloadStatusResponse} from "../../generated";
+import {Module} from "vuex";
+import {actions} from "./actions";
+import {mutations} from "./mutations";
+import {DownloadResultsDependency} from "../../types";
+
+export interface DownloadResultsState {
+    spectrum: DownloadResultsDependency
+    coarseOutput: DownloadResultsDependency
+    summary: DownloadResultsDependency
+}
+
+export const initialDownloadResults = {
+    downloadId: "",
+    downloading: false,
+    statusPollId: -1,
+    status: {} as DownloadStatusResponse,
+    complete: false,
+    error: null
+}
+
+export enum DOWNLOAD_TYPE {
+    SPECTRUM = "Spectrum",
+    COARSE = "CoarseOutput",
+    SUMMARY = "Summary"
+}
+
+export const initialDownloadResultsState = (): DownloadResultsState => {
+    return {
+        spectrum: {...initialDownloadResults},
+        coarseOutput: {...initialDownloadResults},
+        summary: {...initialDownloadResults}
+    }
+}
+
+export const downloadResults: Module<DownloadResultsState, RootState> = {
+    namespaced: true,
+    state: {...initialDownloadResultsState()},
```

Comment:
You could argue that `initialDownloadResults` should be a function like `initialDownloadResultsState` - to similarly help avoid state inadvertently being re-used as it was previously. That would remove need for various instances of spread operator. But that's never a bad idea as a defensive tactic - no need to change.